packages/ui-components/src/components/layout/sidebar/menu/group/Group.vue

```diff
@@ -61,13 +62,14 @@ import Plus from '~~/src/components/global/icon/Plus.vue'
 import Edit from '~~/src/components/global/icon/Edit.vue'
 import ArrowFilled from '~~/src/components/global/icon/ArrowFilled.vue'
 import CommonBadge from '~~/src/components/common/Badge.vue'
+import { ChevronRightIcon } from '@heroicons/vue/24/outline'
 
 defineProps<{
   tag?: string
   title?: string
   collapsible?: boolean
   collapsed?: boolean
-  icon?: 'add' | 'edit'
+  icon?: 'add' | 'edit' | 'view'
```

Comment:
Maybe we can do something like FormButton, `icon?: Nullable<PropAnyComponent>` so it's a bit more flexible, can be a follow-up if cumbersome to do now